src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -865,7 +876,7 @@ subroutine ocn_GM_init(domain, err)!{{{
 
          RediGMinitValue = 1.0_RKIND
          if (config_GM_closure == 'constant') then
-            local_config_GM_lat_variable_c2 = .false.
+            local_config_GM_lat_variable_c2 = .true.
```

Comment:
For constant GM closure, c2 varies with latitude? @vanroekel I'm fine with what you found works best, but could you explain why?

Reply:
this is a holdover from my testing.  I will revert. Thanks for the catch!